### PR TITLE
Add custom statbox for validator stats

### PR DESCRIPTION
**Pull request by hamidra**
This creates custom components for stats on the validator page to prevent the rerenders in the validator page that was happening per each new block due to SessionEra context updates.
If this makes sense we can refactor all other stat boxes by moving them to their own custom component to localize their context update impacts.

### CODE REVIEW

**Comment by rossbulat** [comment]
I'm thinking this `Stats` folder can work project-wide for the refactor

**Comment by hamidra** [comment]
> Works very well! I think it makes sense to move these custom boxes into `pages/Validators/`. Where the `library/` components are intended to be more generic, I see these as more purpose-specific stats intended for the page in question. I will move them in the next commit to show what I am thinking.

re: library being generic it makes sense to move it out of library/, but I also see this stats being reused (e.x. currently "Active era" is shown both in validators and Stake/Active), I believe we can create a folder in the pages/components or /components folder and move them there, to keep it DRY.

**Comment by hamidra** [comment]
>
> > Works very well! I think it makes sense to move these custom boxes into `pages/Validators/`. Where the `library/` components are intended to be more generic, I see these as more purpose-specific stats intended for the page in question. I will move them in the next commit to show what I am thinking.
> 
> re: library being generic it makes sense to move it out of library/, but I also see these stats being reused (e.g. currently "Active era" is shown both in validators and Stake/Active), I believe we can create a folder in the pages/components or /components folder and move them there, to keep it DRY.

I moved them to a Stats subfolder in each page's folder (e.g. ./Validators/Stats). currently, we only reuse ActiveEra stat box which I kept it in Validators/Stats. We can always refactor in the future if we find a better structure works better!
 
Sorry, my formatter messed up the files again a little bit, I'll add a linter to the package.json to have a universal format for the whole repo:)

**Comment by rossbulat** [comment]
> Sorry, my formatter messed up the files again a little bit, I'll add a linter to the package.json to have a universal format for the whole repo:)

My apologies, I extended eslint with prettier yesterday, did a project-wide reformat.

**Comment by rossbulat** [comment]
> re: library being generic it makes sense to move it out of library/, but I also see these stats being reused (e.g. currently "Active era" is shown both in validators and Stake/Active), I believe we can create a folder in the pages/components or /components folder and move them there, to keep it DRY.

I agree with you, I think these re-usable preconfigured stat boxes should be in `library/`.

I will rebase and try to get rid of the conflicts.

File: src/pages/Stake/Active/index.tsx [hunk 1]
```diff
@@ -22,16 +22,16 @@ import { GenerateNominations } from '../GenerateNominations';
 import { PageTitle } from '../../../library/PageTitle';
 import { OpenAssistantIcon } from '../../../library/OpenAssistantIcon';
 import { useModal } from '../../../contexts/Modal';
-import StatBoxListItem from '../../../library/StatBoxList/Item';
-import { useStats } from './stats';
 import { PAYEE_STATUS } from '../../../constants';
+import ActiveNominationsStatBox from './Stats/ActiveNominations';
+import MinimumActiveBondStatBox from './Stats/MinimumActiveBond';
+import ActiveEraStatBox from '../../Validators/Stats/ActiveEra';
```
Comment (rossbulat) [inline src/pages/Stake/Active/index.tsx:28]: Good workaround. I think the ActiveEra is more suited to the staking page than Validators.

Perhaps we can introduce another stat to the validator page, maybe how many validators are validating parachains or something along those lines.

**Review by rossbulat — DISMISSED** [review]

**Review by rossbulat — APPROVED** [review]

**Pull request merged**